.github/workflows/e2e_tests.yml

```diff
@@ -41,29 +27,28 @@ jobs:
       - name: Setup pnpm
         uses: ./.github/actions/pnpm-setup
 
-      - name: Cache Playwright browsers
-        id: playwright-cache
-        uses: actions/cache@v4
-        with:
-          path: ~/.cache/ms-playwright
-          key: playwright-${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml', '**/playwright.config.ts') }}
-          restore-keys: |
-            playwright-${{ runner.os }}-
-
-      - name: Install Playwright browsers
-        if: steps.playwright-cache.outputs.cache-hit != 'true'
-        run: pnpm exec playwright install --with-deps
-
-      - name: Install system dependencies for WebKit
-        # Some WebKit dependencies seem to lay outside the cache and will need to be installed separately
-        if: ${{ steps.playwright-cache.outputs.cache-hit == 'true' }}
-        run: pnpm exec playwright install-deps webkit
-
```

Comment:
@FunamaYukina Sorry, deleting this lines are intentional too?

Reply:
I am so sorry. I will fix it.🙏

Reply:
The following commit corrects the problem and adds a step called Check deployment conditions.🙏
https://github.com/liam-hq/liam/pull/1053/commits/167e62c6df6930ab3c07da313fa4ab7c58e62645 ... revert code
https://github.com/liam-hq/liam/pull/1053/commits/fd7746e3e87e0258bf18bbc4b3ebaef23b0bd56b ... add ``Check deployment conditions`` step 
https://github.com/liam-hq/liam/pull/1053/commits/8028a2969362784cbdc1f03f30979c31db3fc3de , https://github.com/liam-hq/liam/pull/1053/commits/2cb70023c637a49dc6fcbbe406b9c25f15314de9 ... fix codes